c2/c2-client-bundle/c2-client-service/src/main/java/org/apache/nifi/c2/client/service/C2ClientService.java

```diff
@@ -41,8 +41,8 @@ public C2ClientService(C2Client client, C2HeartbeatFactory c2HeartbeatFactory, C
     }
 
     public void sendHeartbeat(RuntimeInfoWrapper runtimeInfoWrapper) {
-        C2Heartbeat c2Heartbeat = c2HeartbeatFactory.create(runtimeInfoWrapper);
-        client.publishHeartbeat(c2Heartbeat).ifPresent(this::processResponse);
+            C2Heartbeat c2Heartbeat = c2HeartbeatFactory.create(runtimeInfoWrapper);
```

Comment:
This indentation seems unnecessary could you please double check?

Reply:
Thanks, fixed it.